.github/workflows/prioritize-by-reactions.yaml

```diff
@@ -0,0 +1,94 @@
+name: Prioritize Issues by Reactions
+
+on:
+  schedule:
+    - cron: '0 0 * * *'   # Runs once a day at midnight UTC
+  workflow_dispatch:       # Allows manual runs
+
+jobs:
+  prioritize:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get install -y jq
+
+      - name: Prioritize issues by reactions
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: microsoft/vsmarketplace
+          BUG_HIGH: 10
+          BUG_MEDIUM: 5
+          FEATURE_MEDIUM: 25
+        run: |
+          issue_numbers=$(gh issue list -R "$REPO" --state open --json number -q '.[].number')
+
+          for ISSUE in $issue_numbers; do
+            echo "🔍 Checking Issue #$ISSUE"
+
+            # Get issue labels
+            labels=$(gh issue view $ISSUE -R $REPO --json labels -q '.labels[].name' | tr '\n' ' ')
+            echo "   Labels: $labels"
+
+            reactions=$(curl -s \
+              -H "Accept: application/vnd.github.squirrel-girl-preview+json" \
+              -H "Authorization: token $GH_TOKEN" \
+              https://api.github.com/repos/$REPO/issues/$ISSUE/reactions)
+
+            total=$(echo "$reactions" | jq 'length')
+
+            # Determine priority based on issue type
+            if echo "$labels" | grep -q "Type:Bug"; then
+              echo "   🐛 Bug issue detected"
+              if [ "$total" -ge "$BUG_HIGH" ]; then
+                priority="Priority:0"
+              elif [ "$total" -ge "$BUG_MEDIUM" ]; then
+                priority="Priority:1"
+              else
+                priority="Priority:2"
+              fi
+              echo "👉 Issue #$ISSUE has $total reactions — Assigning label: $priority"
+            elif echo "$labels" | grep -q "Type:Feature"; then
+              echo "   ✨ Feature request detected"
+              if [ "$total" -ge "$FEATURE_MEDIUM" ]; then
+                priority="Priority:1"
+                echo "👉 Issue #$ISSUE has $total reactions — Assigning label: $priority"
+              else
+                echo "👉 Issue #$ISSUE has $total reactions — Below threshold ($FEATURE_MEDIUM), no priority label assigned"
```

Comment:
I think this would probably be a 'Priority:2' when it's below the 25 threshold?

Reply:
Yeah that part wasn't super clear to me if we want all issues to have priority labels. Maybe I'll ping Joel and see what he was envisioning. Maybe the support issues should have some priority flag as well then?